src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,92 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
+    private static final int columnsNumber = 2;
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
I suggest to implement this method like this:
```suggestion
    public void getStatistic(String fromFileName, String toFileName) {
        List<String> dataFromFile = readFromFile(fromFileName);
        String report = createReport(dataFromFile);
        writeToFile(report, toFileName);
}
```